implementations/php-justinrainbow-json-schema/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "justinrainbow/bowtie-json-schema-test-harness",
+  "description": "These sources contains the test harness implementation for Bowtie",
+  "license": "MIT",
+  "require": {
+    "justinrainbow/json-schema": "dev-master"
```

Comment:
If perchance you're interested in a follow up PR, changing this to instead be a fixed version, (perhaps as soon as you publish one if that's not yet happened?) would be nice, and then relying on Dependabot to update it.

As is, right now the version information shown at https://bowtie.report/#/implementations/php-justinrainbow-json-schema (which is live, congrats!) won't be useful to someone, since it just says `dev-master`, but someone won't know what commit that corresponds to even if it was HEAD at some point in time.